loopy/kernel/data.py

```diff
@@ -841,4 +844,47 @@ def __init__(self, target_name, result_dtypes, arg_dtypes):
 
 # }}}
 
+
+# {{{ Iname class
+
+class Iname(Taggable):
+    """
+    Records an iname in a :class:`~loopy.LoopKernel`.
```

Comment:
Explain a bit more why this class exists (and maybe what an iname is).

Reply:
Gave a bit more context in 4d9b29b.